project/Dependencies.scala

```diff
@@ -1,8 +1,6 @@
 import sbt._
 
 object Dependencies {
```

Comment:
I'd like to try to trim down the dependencies as much as possible. For example, I question whether we need `logbackClassic`, `swaggerUi`, `httpClient`, `unfilteredFilter`, and `googleRpc`, and we should make sure we're using `ficus` and `enumeration` if we're going to keep those.